CHANGELOG.md

```diff
@@ -50,6 +52,57 @@
+| Description | Additional Information | Ticket Number |
+| --- | --- | --- |
+| After a ledger-triggered upgrade of the Sovrin package failed, indy-node upgraded to an incorrect version | | [INDY-2161](https://jira.hyperledger.org/browse/INDY-2161) |
+| One node doesn't catch up after promotion | | [INDY-2222](https://jira.hyperledger.org/browse/INDY-2222) |
+| A Replica may process messages from other Replicas | | [INDY-2248](https://jira.hyperledger.org/browse/INDY-2248) |
+| Up to F nodes are out of consensus after >3 hours of load | | [INDY-2268](https://jira.hyperledger.org/browse/INDY-2268) |
+| A Node may not connect to another Node after restart | | [INDY-2274](https://jira.hyperledger.org/browse/INDY-2274) |
+| Two View Changes happen during master or backup primary demotion | | [INDY-2247](https://jira.hyperledger.org/browse/INDY-2247) |
+
+#### Changes and Additions
+| Description | Additional Information | Ticket Number |
+| --- | --- | --- |
+| Debug: Integrate PBFT viewchanger service into current codebase | | [INDY-2140](https://jira.hyperledger.org/browse/INDY-2140) |
+| Request missing ViewChange messages when receiving NewView | | [INDY-2178](https://jira.hyperledger.org/browse/INDY-2178) |
+| Basic integration tests with a new View Change protocol need to pass | | [INDY-2223](https://jira.hyperledger.org/browse/INDY-2223) |
+| Recover from a situation when View Change is finished on >= N-F of other nodes | | [INDY-2224](https://jira.hyperledger.org/browse/INDY-2224) |
+| A Primary lagging behind a stable chedkpoints should not send NewView | | [INDY-2230](https://jira.hyperledger.org/browse/INDY-2230) |
+| Do not stabilize checkpoint after the view change if a Replica doesn't have this checkpoint | | [INDY-2231](https://jira.hyperledger.org/browse/INDY-2231) |
+| Save PrePrepare's BatchID in audit ledger and restore list of preprepares and prepares on node startup | | [INDY-2235](https://jira.hyperledger.org/browse/INDY-2235) |
+| PBFT View Change Debug: Part 2 | | [INDY-2244](https://jira.hyperledger.org/browse/INDY-2244) |
+| Optimize Propagate logic | | [INDY-2257](https://jira.hyperledger.org/browse/INDY-2257) |
+
+#### Known Issues
+| Description | Additional Information | Ticket Number |
+| --- | --- | --- |
+| All nodes need to select the same primary during view change | | [INDY-2262](https://jira.hyperledger.org/browse/INDY-2262) |
+| A Node missing a View Change may not be able to finish it if NODE txns have been sent | | [INDY-2275](https://jira.hyperledger.org/browse/INDY-2275) |
+| A new node joining the pool during the view change may not be able to start ordering immediately | | [INDY-2276](https://jira.hyperledger.org/browse/INDY-2276) |
+Summary: If there are NODE txns for adding/removing nodes interleaved with View Changes (not any view changes, but a specific subset), then up to F Nodes may not be able to finish view change. This will not affect consensus since N-F nodes will finish the view change.
```

Comment:
It looks like we have to change the summary because of new findings in INDY-2262:

Summary: If there are NODE txns for adding/removing nodes interleaved with View Changes (not any view changes, but a specific subset), then either up to F or all Nodes may not be able to finish view change. Please see the details and conditions when it may happen in INDY-2262.